### PR TITLE
Remove unused assets and optimize loading

### PR DESCRIPTION
**Pull request by ferferga**
Remove unused assets and use SVGs for all the graphic resources used, reducing bandwidth usage and removing the artifacts that appear when the logo, for example, is being downloaded when the app is launched.

### CODE REVIEW

File: src/img/banner.svg [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- ***** BEGIN LICENSE BLOCK *****
+  - Part of the Jellyfin project (https://jellyfin.media)
+  - 
+  - All copyright belongs to the Jellyfin contributors; a full list can
+  - be found in the file CONTRIBUTORS.md
+  - 
+  - This work is licensed under the Creative Commons Attribution-ShareAlike 4.0 International License.
+  - To view a copy of this license, visit http://creativecommons.org/licenses/by-sa/4.0/.
+- ***** END LICENSE BLOCK ***** -->
```
Comment (ferferga) [inline src/img/banner.svg:10]: Shall we remove this block?

We already have it in the original **jellyfin-ux** repo and we might save a few kilobytes.

Comment (YouKnowBlom) [reply src/img/banner.svg:10]: Sounds fine by me

Comment (YouKnowBlom) [reply src/img/banner.svg:10]: Licensing stuff is not my strong side though

Comment (Camc314) [reply src/img/banner.svg:10]: I'm not sure, I think someone on the general matrix chat mentioned that we should have licensing at the start of all the files. Hopefully someone with more knowledge of licensing can help.

Comment (MrTimscampi) [reply src/img/banner.svg:10]: Ideally, the SVG files should be optimized during build, which would remove the licensing block anyway.

I don't know the build process for this repo, but I'm guessing it's packing everything with Webpack, so using something like [image-webpack-loader](https://github.com/tcoopman/image-webpack-loader), which uses `imagemin` (the same tool we use to optimize images on `jellyfin-web`) should do the trick.

Comment (ferferga) [reply src/img/banner.svg:10]: @MrTimscampi This sounds perfect! I'll investigate this.

Comment (joshuaboniface) [reply src/img/banner.svg:10]: We should leave it, its an SVG which is a source file.

**Comment by ferferga** [comment]
@jellyfin/web Not sure if this is the best method for optimizing images using webpack (I'm a noob with webpack and no way I get file-loader and rest of goodies to work), feel free to commit and improve this. 

It works™ at least :P

File: src/img/banner.svg (hunk 1 above)
Comment (ferferga) [reply src/img/banner.svg:10]: @joshuaboniface Leaving and removing the blocks in production in 24ede02

Comment (EraYaN) [reply src/img/banner.svg:10]: ShareAlike also mandates that any derivative is also properly licensed under the same license so it should be noted on the license/about page. Cause even the minified SVG is STILL CC-BY-SA.

Comment (ferferga) [reply src/img/banner.svg:10]: @EraYaN Adding a header at the bottom  of the LICENSE.md file with a copy-paste of the license included in the SVG is enough?

Sorry, I'm not a license expert

**Comment by YouKnowBlom** [comment]
LGTM. Let me know when the licensing stuff is figured out. 

@MrTimscampi you might want to take a look at this as you're better at webpack than me :p.

File: src/img/banner.svg (hunk 1 above)
Comment (MrTimscampi) [reply src/img/banner.svg:10]: You can pass options to the SVGO plugin of Imagemin to keep comments. The option in question is `removeComments` (`true` by default)

File: webpack.config.js [hunk 2]
```diff
@@ -1,22 +1,38 @@
...
+        }]),
+        new ImageminPlugin({
+            bail: false,
+            cache: false,
+            imageminOptions: {
+                plugins: [
+                    ["svgo", {
+                        plugins: [
+                            {
+                                removeViewBox: false
```
Comment (MrTimscampi) [inline webpack.config.js:29]:
```suggestion
                                removeComments: false,
                                removeViewBox: false
```

File: webpack.config.js [hunk 3]
```diff
@@ -1,22 +1,38 @@
...
     context: path.resolve(__dirname, "src"),
     entry: "./app.js",
     output: {
         filename: "bundle.js",
-        path: path.resolve(__dirname, "dist"),
+        path: path.resolve(__dirname, "dist")
     },
     plugins: [
         new CleanWebpackPlugin(),
         new CopyWebpackPlugin([{
```
Comment (MrTimscampi) [inline webpack.config.js:16]: This should probably be replaced by proper injection at some point for cache busting. More info here: https://webpack.js.org/guides/output-management/

File: webpack.config.js [hunk 4]
```diff
@@ -1,22 +1,38 @@
 const path = require("path");
 const CopyWebpackPlugin = require("copy-webpack-plugin");
 const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const ImageminPlugin = require("imagemin-webpack");
 const webpack = require("webpack");
```
Comment (MrTimscampi) [inline webpack.config.js:5]: Unused dependency

File: webpack.config.js (hunk 2 above)
Comment (MrTimscampi) [reply webpack.config.js:29]: This will ensure that the license is kept.

Comment (EraYaN) [reply webpack.config.js:29]: This is a good change

**Comment by hawken93** [comment]
I have rebased this over at https://github.com/hawken93/jellyfin-chromecast/tree/optimize-loading

File: src/img/banner.svg (hunk 1 above)
Comment (hawken93) [reply src/img/banner.svg:10]: Went with removeComments=false

**Review by hawken93 — APPROVED** [review]
LGTM

**Pull request merged**